src/access_nri_intake/cli.py

```diff
@@ -214,9 +215,12 @@ def _get_project(path):
     # Save the catalog
     cm.save()
 
-    _here = os.path.abspath(os.path.dirname(__file__))
     if update:
-        with open(os.path.join(_here, "data", "catalog.yaml"), "w") as fobj:
+        with (
+            importlib.resources.files("access_nri_intake")
+            .joinpath("data/catalog.yaml")
+            .open(mode="w") as fobj
+        ):
```

Comment:
This change does not get touched by the test suite, and I haven't been able to manually test it... A look at it would be appreciated.

Reply:
@jo-basevi do you have any suggestions for how this might be tested?

Reply:
Sorry I don't have any good ideas on testing. Could have a helper function in the upper-level `utils.py` file return `importlib.resources.files("access_nri_intake").joinpath("data/catalog.yaml")` so that can be reused and that can be tested? Might be worth having global vars for  the "access_nri_intake" package name and the filenames.

I didn't know about `importlib.resources` so that was great to learn about!

Reply:
My main concern is the last line of this block, the `mode="w"`. I'm not sure how the system will like trying to write back to something in its own `data` directory, although I suspect as this is a standard-ish file open, it's probably OK? I might just have to rig up a way to manually test that the write-back is respected, and then test on the catalog location finding only.